code/game/turfs/turf.dm

```diff
@@ -162,6 +162,10 @@
 		var/obj/O = AM
 		if(O.is_frozen)
 			O.make_unfrozen()
+	for(var/I in contents)
+		if(istype(I, /obj/item/device/pressure_plate))
```

Comment:
It's in the turf's contents, which means it's on the floor, which means its ```Crossed()``` proc will be called when walked over.
Remove this code in this file, and simple do:
```
/obj/item/device/pressure_plate/Crossed(atom/movable/AM)
    stepped_on(AM)
```